etc/picongpu/hemera-hzdr/gpu.tpl

```diff
@@ -63,7 +63,7 @@
 
 # number of cores to block per GPU - we got 6 cpus per gpu
 #   and we will be accounted 6 CPUs per GPU anyway
-.TBG_coresPerGPU=6
```

Comment:
The system contains `2x Intel Xeon Gold 6136, 12 Cores/CPU` therefore 6 is correct.

Reply:
Yes, that was exactly my point. But it just does not work like that somehow.